EventsExpress/ClientApp/src/components/helpers/validators/event-validate-form.js

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
why does this one not in the same folder where it is used? or have we a few such places?